tile/targets/cpu/jit.cc

```diff
@@ -1683,6 +1721,8 @@ llvm::JITSymbol Runtime::findSymbol(const std::string& name) {
       {"_libxsmm_smmdispatch", symInfo(libxsmm_smmdispatch)},
       {"libxsmm_dmmdispatch", symInfo(libxsmm_dmmdispatch)},
       {"_libxsmm_dmmdispatch", symInfo(libxsmm_dmmdispatch)},
+      {"RunTimeLogEntry", symInfo(rt::RunTimeLogEntry)},   // For debugging
+      {"_RunTimeLogEntry", symInfo(rt::RunTimeLogEntry)},  // For debugging
```

Comment:
Is the second entry really necessary? There is logic a few lines down which is intended to handle the platform-dependent leading "_" character.

Reply:
It was necessary. I tried without it and the linker couldn't find the method to link with. So, I added the second entry.